onnx/defs/nn/defs.cc

```diff
@@ -858,17 +881,29 @@ ONNX_OPERATOR_SET_SCHEMA(
             "in effect, the operation expects input data tensor "
             "to arrive with the dimension denotation of [DATA_BATCH, "
             "DATA_CHANNEL, DATA_FEATURE, DATA_FEATURE ...].",
-            "T1")
+            "T1",
+            OpSchema::Single,
+            true,
+            1,
+            OpSchema::Differentiable)
```

Comment:
The tags for quantized ops make sense. Just out of curiosity, are they also differentiable in Pytorch?

Reply:
Torch indicates these quantized variables as Differentiable